src/iop/overexposed.c

```diff
@@ -415,34 +381,14 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
     _MM_SET_FLUSH_ZERO_MODE(oldMode);
   #endif
 
+process_finish:
   if(piece->pipe->mask_display & DT_DEV_PIXELPIPE_DISPLAY_MASK)
     dt_iop_alpha_copy(ivoid, ovoid, roi_out->width, roi_out->height);
 
-  dt_free_align(img_tmp);
+  if(img_tmp != NULL) dt_free_align(img_tmp);
```

Comment:
`dt_free_align()` should be a no-op if `img_tmp == NULL`

Reply:
Yes, indeed. Changed.